feature/system/management/otg_tests/management_ha_test/management_ha_test.go

```diff
@@ -54,8 +54,24 @@ var (
 		IPv4Len: 32,
 		IPv6Len: 128,
 	}
-
-	mgmtVRF  = "mvrf1"
+	mgmtVRF = map[ondatra.Vendor]string{
+		ondatra.JUNIPER: "mgmt_junos",
```

Comment:
Hi Mohana, as we are using this vrf for device reachability, we are looding access to device in middle of test. can we use a different vrf name here please

Reply:
Hi Swetha, Updated the vrf name.Please check.